3rdparty/libcxx/libcxx/src/locale.cpp

```diff
@@ -5642,6 +5642,9 @@ __init_pat(money_base::pattern& pat, basic_string<charT>& __curr_symbol_,
                 return;
             case 1:  // Space between currency-and-sign or currency and value.
                 pat.field[1] = none;
+
```

Comment:
Why the change to this file?   Offhand, looks like it should be reverted

Reply:
This is probably done by format-code. I will revert.